prisma/schema.prisma

```diff
@@ -11,14 +11,111 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Log {
-  id      String @id @default(uuid())
-  level   Level
-  message String
+model Ship {
+  id      String  @id @default(uuid())
+  name    String  @unique
+  // maneuvers Maneuver[] // I don't know how we want to handle custom types here, because what I want is a {ManeueverType, Difficulty} tuple, e.g. {"5K", "Red"}
+  // If we ever hit the case of a chassis having different hull/shields (1.0 YT-1300), we'd have to move these to the pilot
+  hull    Int
+  shields Int
+  // arcs Arc[] // Same as maneuevers - I want tuples and I don't know the right way to do it with typing, e.g. [{"front", "2'}, {"bullseye", "3'}]
+  pilots  Pilot[]
 }
 
-enum Level {
-  Info
-  Warn
-  Error
+model ShipAbility {
+  id    String  @id @default(uuid())
+  name  String
+  text  String
+  Pilot Pilot[]
+}
+
+model Pilot {
```

Comment:
Do you want a field for flavor text? Unsure if it's exclusion is on purpose or not (or if I'm blind)

Reply:
...........good question. It's so easy that it should probably exist if I'm not sure. Will add it!